drivers/volume/portworx/portworx.go

```diff
@@ -980,6 +1088,25 @@ func (d *portworx) pickAlternateClusterManager(n node.Node) (api.OpenStorageNode
 	return nil, fmt.Errorf("failed to get an alternate cluster manager for %s", n.Name)
 }
 
+// EnableStorageExpansion creates autopilot rules
+func (d *portworx) EnableStorageExpansion(apRules []apapi.AutopilotRule) error {
+	for _, apRule := range apRules {
+		autopilotRule, err := d.schedOps.CreateAutopilotRule(apRule)
+		if err != nil {
+			return err
+		}
+		d.autopilotRules = append(d.autopilotRules, autopilotRule)
+	}
+	return nil
+}
+
+func (d *portworx) IsStorageExpansionEnabled() bool {
```

Comment:
This is incorrect. 

Just because you have autopilotRules does not mean storage expansion is enabled.

What do we do when an autopilot rule has been written for a different use case?